cypress/e2e/tests/administration/custom-migration-targets/field_validation.test.ts

```diff
@@ -113,18 +113,12 @@ describe(["@tier1"], "Custom Migration Target Validations", () => {
         CustomMigrationTarget.selectRepositoryType(RepositoryType.git);
 
         CustomMigrationTarget.fillRepositoryUrl("   ");
-        cy.get(CustomMigrationTargetView.repositoryUrlHelper).should(
-            "contain",
-            "Must be a valid URL."
-        );
+        cy.get(CustomMigrationTargetView.helperText).should("contain", "Must be a valid URL.");
```

Comment:
We have function `doesExistText()` for such things, or if it doesn't match your needs please move it to a separate function in utils (`doesObjectContainText()` for example) as this can be reused multiple times.

Reply:
done